pkg/ensurance/analyzer/analyzer.go

```diff
@@ -483,7 +483,7 @@ func (s *AnormalyAnalyzer) merge(stateMap map[string][]common.TimeSeries, avoida
 			}
 
 			if action.Spec.Eviction != nil {
-				var deletionGracePeriodSeconds = utils.GetInt32withDefault(action.Spec.Eviction.TerminationGracePeriodSeconds, executor.DefaultDeletionGracePeriodSeconds)
+				var deletionGracePeriodSeconds = utils.GetUint32withDefault(action.Spec.Eviction.TerminationGracePeriodSeconds, executor.DefaultDeletionGracePeriodSeconds)
```

Comment:
GetGracePeriodSeconds is better.

Reply:
GetInt32withDefault is a generic function for many scenarios such as get piority（utils.GetInt32withDefault(pod.Spec.Priority, 0)）.  

If we rewrite the code like below, it will be too complex.

var deletionGracePeriodSeconds = utils.GetGracePeriodSeconds(action.Spec.Eviction.TerminationGracePeriodSeconds, executor.DefaultDeletionGracePeriodSeconds)

func GetGracePeriodSeconds (i *int32, value int32) int32 {
	return GetInt32withDefault(i,value)
}

func GetInt32withDefault(i *int32, value int32) int32 {
	if i != nil {
		return *i
	}

	return value
}